### ISSUE TITLE
Scaphoid bone fracture

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10672]
**Preferred term label:**
Scaphoid bone fracture

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Scaphoid fracture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001107
- label: Fractured scaphoid bone
- parents: [Fractured carpal bone (HP:0041248)](https://hpo.jax.org/app/browse/term/HP:0041248)
- pmid: PMID:24298746
- def: A partial or complete breakage of the scaphoid bone.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10672
- synonyms: Scaphoid bone fracture(NONE)|Scaphoid fracture(NONE)
